### DISCUSSION TITLE
2nd Emporia Vue flashed with ESPHome.  Need only CT readings.

### DISCUSSION THREAD

**Discussion by weapons** [emporia-vue-local/esphome#243] [Q&A]
Hi Everyone.  

I have a two-story home with separate Consumer Units for each floor.  I have already installed an Emporia Vue flashed with ESPHome on the 1st floor (main box) and it's working fine.  I installed a second Emporia Vue also flashed on the second floor but did not install the Phase A,B,C main clamps as these are already monitored by the 1st unit.  With the second unit (upstairs) I only want to monitor the 16 circuits Power and Daily Energy.   I installed the sample config just like the 1st unit but all my circuit readings show 0W.  I guess the reason for this is that the config references the 3 Phases which are not being monitored as those clamps are not attached.

Q1. Is there a way for the second unit to reference the 3 Phase (A,B,C) readings from the first unit?  

Q2. Is it necessary to reference the 3 main Phases at all since I just want to monitor individual circuit's Power consumption?

Thanks in advance.

**Comment by ajacques** [comment]
> Is there a way for the second unit to reference the 3 Phase (A,B,C) readings from the first unit?

It's not required to install the phase A, B, & C CT camps in the second unit, however you do you need to connect the Phase A+B+C electrical connections. The Emporia Vue samples the frequency and voltage quite rapidly as part of it's power usage calculations and it wouldn't be able to send the readings over the network to the second Vue device. In addition, this calculation is done by the onboard MCU, not the esp32.

> Is it necessary to reference the 3 main Phases at all since I just want to monitor individual circuit's Power consumption?

I think what you could do is attach to the 2 or 3 phases electrical connectors inside the electrical box, but then mark the phases as `internal: true` so they don't get sent to MQTT/Home Assistant and you can minimize the data usage.

**Reply by weapons** [reply]
Hi. I'm really sorry for the late reply. I didn't realize I didn't have notifications turned off for replies. Anyways, I connected as you mentioned and wirks like a charm. Thank you very much.

**Comment by beardymarrow** [comment]
Hi, no you don't need to attach the phase a, b or c clamps. So long as you have all 3 phase wires from the bottom plug connected to the relevant phase you'll be fine, as that's where it measures voltage, frequency and phase from.
Just comment out the clamp config.

**Reply by weapons** [reply]
Hi. Sorry for the late reply. Thank you very much. Just connected the 3 input wires to their respective phases and all working great.
